src/ActionsTestResultAction/TestResultCollection.cs

```diff
@@ -55,11 +55,14 @@ public string Format(string? title, TestResultFormatMode mode, int maxLen, bool
 
             var testId = 0;
 
+            var s = useEmojis ? " 💤" : string.Empty;
+            var p = useEmojis ? " ✅" : string.Empty;
+            var f = useEmojis ? " ❌" : string.Empty;
             _ = mb.AppendLine($"""
 
-            | | Total | Skipped | Passed | Failed |{errorTableHdrExt2}
+            | | Total | Skipped | Passed  | Failed |{errorTableHdrExt2}

```

Comment:
Where did this extra space come from? Does it affect the look of the layout?

Reply:
I've added the emojis to the title, forgot to properly cleanup